Automation/Redis_Cache/deploy.ps1

```diff
@@ -0,0 +1,27 @@
+
+# to run this, navigate to the Powershell  /Automation/Redis_Cache/deploy.ps1
+
+
+param(
+        [string] [Parameter(Mandatory=$true)] $SubscriptionId,
+        [string] [Parameter(Mandatory=$true)] $ResourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $Location,
+        [string] [Parameter(Mandatory=$true)] $Redis_nsc_redis_dev_usw2_thursday_name
+
+      )
+
+$pathToRedisTemplate = "/Users/student/Documents/GitHub/ad440-winter2021-thursday-repo/Automation/Redis_Cache/template.json"   
+
+
+
+# Creates REDIS CACHE if one of the same name does not already exist in the Resource Group
+$RedisExists = (Get-AzRedisCache -ResourceGroupName “nsc-rg-dev-usw2-thursday” -Name “nsc-redis-dev-usw2-thursday”) 
+      
```

Comment:
Could you use the initial parameters for these arguments, instead of hard coding the values?